capstone-frontend/src/components/ClubSearchList.jsx

```diff
@@ -15,60 +15,60 @@ export class ClubSearchList extends Component {
 
   getData = async () => {
     // Get clubs by searchQuery
-    let clubs = await fetch(`/api/clubSearch?searchTerm=${this.props.searchQuery}`)
+    let clubs = await fetch(`/api/clubs?searchTerm=${this.props.searchQuery}`)
       .then(response => response.json())
       .catch(function (err) {
         alert(err)
       });
 
     // Get clubs by checking to see if any clubs are reading the books found in Google Books API search
-    await Promise.all(this.props.books.map(async (book) => {
-      const clubsWithBook = await fetch(`/api/clubs?gbookID=${book.id}`)
+    await Promise.all(this.props.books.map(book => {
+      return fetch(`/api/clubs?gbookID=${book.id}`)
         .then(response => response.json())
+        .then(clubsWithBook => {
+          if (clubsWithBook > 0) {
```

Comment:
Should this me `clubsWithBook.length`? (assuming this is an array)

Reply:
Yes it should be! Must've gotten missed in the translation